Proyecto/src/main/java/com/example/Proyecto/dto/Formulario.java

```diff
@@ -6,6 +6,6 @@
 @Data
 @ToString
 public class Formulario {
-    private String nombre;
+    // private String nombre;
     //private String otrodato;
```

Comment:
I know It's not useful but for @fernandondin @axelthepony27 @caams  Do you think it's necessary to block completely, Wouldn't be better if we just erase this file, would be?

Reply:
Podemos dejarla, igual creo que tendremos que hacer el formulario en las siguientes prácticas. Comenté esa línea porque en general no estamos usando las variables de esa clase.